tensorboard/components/ng_polymer_lib.html

```diff
@@ -14,6 +14,7 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<link rel="import" href="security.html" />
```

Comment:
Does this always have to be imported first?  If so, maybe worth a comment.

Reply:
Yes and done!